ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/management-security.j2

```diff
@@ -63,6 +63,32 @@ management security
 {%             endif %}
 {%         endfor %}
 {%     endif %}
+{%     for profile in management_security.shared_secret_profiles | arista.avd.natural_sort("profile") %}
+   session shared-secret profile {{ profile.profile }}
+{%         for secret in profile.secrets | arista.avd.natural_sort("name") %}
+{%             if secret.secret is arista.avd.defined %}
+{%                 set secret_cli = "secret " ~ secret.name ~ " " ~ secret.secret_type | arista.avd.default("7") ~ " " ~ secret.secret | arista.avd.hide_passwords(hide_passwords) %}
```

Comment:
```suggestion
{%                 set secret_cli = "secret " ~ secret.name ~ " " ~ secret.secret_type | arista.avd.default("0") ~ " " ~ secret.secret | arista.avd.hide_passwords(hide_passwords) %}
```

since 7 requires encrypted value in secret.name. 
Correct me if I am wrong.

Reply:
Fixed.